tasktiger/lua/move_task.lua

```diff
@@ -0,0 +1,95 @@
+local function zadd_w_mode(key, score, member, mode)
```

Comment:
"nx", "min", and "" are the only ones we actually use, right? So we could technically skip including "xx" and "max".

Reply:
true, will trim it